examples/khr-3hv/khr-3hv_continuous/controllers/supervisorsManager/supervisorsManager.py

```diff
@@ -0,0 +1,46 @@
+from Args import ModelArgs
+import PPORun
+
+"""
+Modify these constants if needed.
```

Comment:
No need to refer and define as 'constants'.
```suggestion
Modify these parameters if needed.
```